Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,53 @@
 import React from 'react';
 import './Display.css';
+import NumberButton from '../ButtonComponents/NumberButton';
+import ActionButton from '../ButtonComponents/ActionButton';
+import ClearButton from '../ButtonComponents/ClearButton';
+import Zero from '../ButtonComponents/Zero';
+import OutputDisplay from '../OutputDisplay';
+
+const CalculatorDisplay = props => {
+  return (
+    <div className="calculator">
+    <div className="buttons">
+      <div className="numButtons">
+        <div className="zero-btn">
+          <Zero />
+        </div>
+        <div className="numRowOne">
+          <NumberButton number="1"/>
+          <NumberButton number="2" />
+          <NumberButton number="3" />
+        </div>
+        <div className="numRowTwo">
+          <NumberButton number="4" />
+          <NumberButton number="5" />
+          <NumberButton number="6" />
+        </div>
+        <div className="numRowThree">
+          <NumberButton number="7" />
+          <NumberButton number="8" />
+          <NumberButton number="9" />
+        </div>  
+        <div className="clear-btn">
+          <ClearButton />
+        </div>
```

Comment:
you can also pass props here and add `action="clear"` if you want to pass props down on ClearButton so it's not hard-coded in that component